src/ir_Mitsubishi.cpp

```diff
@@ -1423,19 +1400,17 @@ void IRMitsubishi112::setSwingV(const uint8_t position) {
     case kMitsubishi112SwingVHigh:
     case kMitsubishi112SwingVHighest:
     case kMitsubishi112SwingVAuto:
-      setBits(&remote_state[kMitsubishi112SwingVByte],
-              kMitsubishi112SwingVOffset, kMitsubishi112SwingVSize, position);
+      _.SwingV = position;
       break;
     default:
-      setMode(kMitsubishi112SwingVAuto);
```

Comment:
and this

Reply:
agreed. Some cut & paste fail. Same notes as per previous instance of the mistake/bug.